src/linker/Linker.Steps/BaseSubStep.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Mono.Cecil;
+
+namespace Mono.Linker.Steps
+{
+	public abstract class BaseSubStep : ISubStep
+	{
+		public AnnotationStore Annotations => Context.Annotations;
```

Comment:
Should this be `protected` just like the `Context` - it seems weird those two are not the same.

Reply:
Resolved